example/src/APIHandler.js

```diff
@@ -40,12 +40,14 @@ const getAxiosInstance = () => {
 // creates a user, https://docs.getphyllo.com/docs/api-reference/b3A6MTQwNjEzNzY-create-a-user
 export const createUser = async (name, externalId) => {
   try {
+    debugger
     const api = getAxiosInstance()
-
+    console.log(`createUser : ${api}`)
```

Comment:
We can remove these console statements

Reply:
@chiragphyllo ok sure where i have found console.log